### PR TITLE
#167728005 Configure Eslint

### PR DESCRIPTION
**Pull request by Kingobaino**
#### What does this PR do?
The configuration helps to find problematic pattern or codes that does not follow our style guide
#### Description of Task to be completed?
Have the Eslint configured to follow the following style guide
- Airbnb style guide
- React plugin
- Import plugin
- And set of  other Rules
#### How should this be manually tested?
By adding console.log() in the code or by not  giving an empty line between class members.
#### What are the relevant pivotal tracker stories?
[#167728005](https://www.pivotaltracker.com/n/projects/2381984/stories/167728005)

### CODE REVIEW

**Comment by biolabalo** [comment]
* Your PR description has some spelling errors and  the `What are the relevant pivotal tracker stories?`  isn't properly written

**Comment by Abobos** [comment]
I think you should add a screenshot to your PR @Kingobaino

File: .eslintrc.json [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+    }],
+    "require-jsdoc": ["error", {
+        "require": {
+            "FunctionDeclaration": true,
+            "MethodDefinition": true,
+            "ClassDeclaration": true
+        }
+    }]
+    }
+}
```
Comment (dieudonneAwa) [inline .eslintrc.json:47]: Hey @Kingobaino, you should fix this line. There was supposed to be an empty line after this closing bracket

**Comment by Wokoro** [comment]
> I think you should add a screenshot to your PR @Kingobaino

Please screenshots to show what?

**Comment by Abobos** [comment]
@Wokoro, screenshot to show that eslint is in action.

**Pull request closed without merging**